index file

LouisJarrett opened the pull request:
Updated the index file with headings, input box and submit button

Commit:
index file

Updated the index file

tanyapowell commented on index.html:
```diff
@@ -7,7 +7,17 @@
-
+    <h1>The Black Binder</h1>
+    <h2>Your number 1 directory for Black Culture</h2>
+    <br/>
```
No need to use `<br>`. All spacings can be done within CSS

LouisJarrett replied:
This has now been removed

tanyapowell commented on index.html:
```diff
@@ -7,7 +7,17 @@
+    <h2>Your number 1 directory for Black Culture</h2>
+    <br/>
+    <p>Enter your email address and we will let you know when we are almost ready to go live</p>
+    <form action="/action_page.php">
```
Are we using php? If so, we'll need a server

LouisJarrett replied:
No we are not using PHP. I have removed the form tag

tanyapowell commented on index.html:
```diff
@@ -7,7 +7,17 @@
+    <p>Enter your email address and we will let you know when we are almost ready to go live</p>
+    <form action="/action_page.php">
+        Email: <input type="text" name="Email" value="">
+        <input type="submit" value="Submit">
```
Should be a `<button>`

tanyapowell replied:
`value` should be **Notify Me**

LouisJarrett replied:
Have changed to button....also changed the value as requested.

tanyapowell commented on index.html:
```diff
@@ -7,7 +7,17 @@
+    <br/>
+    <p>Enter your email address and we will let you know when we are almost ready to go live</p>
+    <form action="/action_page.php">
+        Email: <input type="text" name="Email" value="">
```
Why is **Email:** outside of the `<input>`?

Can you write something in the `value` for the placeholder?

LouisJarrett replied:
Email has been removed. I have added "Your email address" in the placeholder

tanyapowell commented on index.html:
```diff
@@ -7,7 +7,17 @@
+        Email: <input type="text" name="Email" value="">
+        <input type="submit" value="Submit">
+    </form>
+    <br/>
```
See above

LouisJarrett replied:
done

tanyapowell commented on index.html:
```diff
@@ -7,7 +7,17 @@
+    </form>
+    <br/>
+    <br/>
+    <i>Twitter</i><i>FaceBook</i><i>Instagram</i>
```
These should be font awesome icons, as per the wireframes.

Also, they need to link to their social media platforms

tanyapowell commented on index.html:
```diff
@@ -7,7 +7,17 @@
 <body>
-
+    <h1>The Black Binder</h1>
+    <h2>Your number 1 directory for Black Culture</h2>
```
Looks weird saying **number 1**, either have it **no. 1** or **number one**. Something doesn't seem right with this

LouisJarrett replied:
changed to number one

tanyapowell reviewed (changes requested):
Can you add a screengrab of what it looks like please? Ta 😃

tanyapowell commented:
@LouisJarrett have you considered accessibility? If not, I'll open up an issue and assign it to you

Commit:
index file

Have made amendments to headings, input box, button and social media
icons

tanyapowell commented on index.html:
```diff
@@ -1,13 +1,23 @@
 
 <html>
-
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
`font-awesome` is already in the `master` branch and should be inside the `<head>` tag. 
Can you remove this

LouisJarrett replied:
This has been removed

tanyapowell commented on index.html:
```diff
@@ -1,13 +1,23 @@
 
+    <input type="text" name="Email" value="Your email address...">
+    <input type="button" value="Notify Me">
+<div class="socialMedia">
```
Can you indent please

LouisJarrett replied:
i have indented

tanyapowell commented on index.html:
```diff
@@ -1,13 +1,23 @@
+    <p>Enter your email address and we will let you know when we are almost ready to go live</p>
 
+    <input type="text" name="Email" value="Your email address...">
+    <input type="button" value="Notify Me">
```
can you change this from `<input>` to `<button>`?

LouisJarrett replied:
done

Commit:
Changed to button type, indenting code and removed link to font awesome

The button has been changed to the type of button. Some indenting has
been applied. The font awesome link has been removed.

tanyapowell commented on index.html:
```diff
@@ -7,7 +7,17 @@
+    <h1>The Black Binder</h1>
+    <h2>Your number one directory for Black Culture</h2>
+    <p>Enter your email address and we will let you know when we are almost ready to go live</p>
 
```
blank line?

LouisJarrett replied:
removed

tanyapowell replied:
is it?

ghost commented:
No, I have not considered accessibility

Commit:
Removed blank lines

removed all the blank lines in the file

Commit:
Merge branch 'master' into Louis
